core/domain/feedback_services.py

```diff
@@ -94,6 +94,18 @@ def create_thread(
         exploration_id, state_name, original_author_id, subject, text, False)
 
 
+def add_summary_to_feedback(thread_id, summary):
```

Comment:
Avoid creating entire functions in main code that are used only by tests. That's an antipattern.